states/MainMenu/MainMenu.tscn

```diff
@@ -125,32 +125,66 @@ The above copyright notice and this permission notice shall be included in all c
 THE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE."
 autowrap = true
 
-[node name="FreeTypeTitle" type="Label" parent="CreditsPane/VBoxContainer/PanelContainer/ScrollContainer/VBoxContainer"]
+[node name="CronAudioTitle" type="Label" parent="CreditsPane/VBoxContainer/PanelContainer/ScrollContainer/VBoxContainer"]
 margin_top = 403.0
 margin_right = 580.0
 margin_bottom = 437.0
+text = "Cron Audio"
+align = 1
+
+[node name="CronAudio" type="Label" parent="CreditsPane/VBoxContainer/PanelContainer/ScrollContainer/VBoxContainer"]
```

Comment:
Move all this stuff to the top of the credits, so all of the Godot engine and library licenses are together